### ISSUE TITLE
Implement the new AdwAboutDialog

### ISSUE THREAD

**Issue by prateekmedia** [gtk-flutter/libadwaita#68]
Design:
![about-dialog-detailed](https://user-images.githubusercontent.com/41370460/179157887-dc1ec891-c22a-4c10-b1cb-fe77ab9c0813.png)

**Comment by simrat39** [comment]
I can take this on, but from my testing on the adwaita test app, this is just a new window, not a dialog which cant leave the app, so I'm not sure if we should implement it as a dialog or not

**Comment by prateekmedia** [comment]
Yes but implementing it as a Window would not be a easy task may need to use multi window packages from pub.dev

**Comment by simrat39** [comment]
Yeah ill just stick to a normal dialog, don't think anyone wants to move it out of the main window anyways

**Comment by simrat39** [comment]
Looks like we already have something similar https://github.com/gtk-flutter/libadwaita/blob/main/lib/src/widgets/adw/about_window.dart, or is there any difference?

**Comment by prateekmedia** [comment]
We already have but it's not that feature rich, as It was create before the about window was merged.

Like we don't have credits, legal or acknowledgement section as shown in above Gnome Builder pics.

Although we don't need that but we need some kind of way for users to add it

Also current about dialog doesn't have animation when navigating inside the dialog

**Comment by simrat39** [comment]
Ok sounds good, so I just extend that widget then right

**Comment by prateekmedia** [comment]
Yes that would work

**Comment by simrat39** [comment]
![adw_aboot_demo](https://user-images.githubusercontent.com/37233846/183589763-435e081c-bff0-4cc6-b865-0d37fb4cb455.gif)

Got this much done, will continue working on it tomorrow

**Comment by prateekmedia** [comment]
Wow, looks great. Animation look cool.

**Comment by Henry-Hiles** [comment]
> ![adw_aboot_demo](https://user-images.githubusercontent.com/37233846/183589763-435e081c-bff0-4cc6-b865-0d37fb4cb455.gif) [ ![adw_aboot_demo](https://user-images.githubusercontent.com/37233846/183589763-435e081c-bff0-4cc6-b865-0d37fb4cb455.gif) ](https://user-images.githubusercontent.com/37233846/183589763-435e081c-bff0-4cc6-b865-0d37fb4cb455.gif) [ ](https://user-images.githubusercontent.com/37233846/183589763-435e081c-bff0-4cc6-b865-0d37fb4cb455.gif)
> 
> Got this much done, will continue working on it tomorrow

looks very cool, is this done now?
